lib/grape/middleware/formatter.rb

```diff
@@ -19,6 +19,7 @@ def before
       end
 
       def after
+        return unless @app_response
```

Comment:
Raised an exception if your suggestion is reflect.
Should the formatter's after method fully be executed in this case?